external/source/rdll_template/README.md

```diff
@@ -0,0 +1,27 @@
+# Reflective DLL Project Template
+This is a project template for Microsoft's Visual Studio to aid in the creation
+of Reflective DLLs.
+
+## Installation
+To install the template, copy the template file to the Visual Studio templates
+directory. For a default Visual Studio 2019 installation the command would be:
+
+```
+copy "Reflective DLL.zip" "%USERPROFILE%\Documents\Visual Studio 2019\Templates\ProjectTemplates"
+```
+
+After the template has been copied, restart Visual Studio and select "Reflective
+DLL" from the available project templates.
+
+Ensure that the [ReflectiveDLLInjection][1] submodule has been cloned and is up
+to date with: `git init submodule; git submodule update`.
```

Comment:
Only thought is to provide some output here so people can verify what they should be seeing when these commands are run, as these commands aren't going to show where stuff is being cloned from which is important for documentation and in case we change the way this works in the future.

Reply:
Hmm ran these commands on my host but for some reason they didn't work this time around:

```
*censored*:*censored*/metasploit/metasploit-framework$ git init submodule
Initialized empty Git repository in /mnt/c/*censored*/metasploit/metasploit-framework/submodule/.git/
*censored*:/mnt/c/*censored*/metasploit/metasploit-framework$  git submodule update
*censored*:/mnt/c/*censored*/metasploit/metasploit-framework$
```

Reply:
Should probably also add verification steps to the effect of `to confirm that this command ran successfully one can also check the external\source\ReflectiveDLLInjection directory to ensure XXX files have been placed in the directory`, as a secondary way of checking things

Reply:
Reviewing things I think this should be `git submodule init`. Other command should be fine

Reply:
And here is output for reference:

```
$ git submodule init
Submodule 'external/source/ReflectiveDLLInjection' (https://github.com/rapid7/ReflectiveDLLInjection.git) registered for path 'external/source/ReflectiveDLLInjection'
$ git submodule update
Cloning into '/mnt/c/*censored*/metasploit/metasploit-framework/external/source/ReflectiveDLLInjection'...
Submodule path 'external/source/ReflectiveDLLInjection': checked out '88e8e5f109793f09b35cb17a621f33647d644103'
$
```

Reply:
Did not know about using angle brackets for Markdown formatting, but really loving the new additions. Thanks @zeroSteiner!